src/test/regress/Makefile

```diff
@@ -191,7 +191,7 @@ check-isolation-base: all  $(isolation_test_files)
 
 # ci takes regression.diffs output from another location, so copy diffs file at the end.
 check-vanilla: all
-	$(pg_regress_multi_check) --vanillatest || (cp $(vanilla_diffs_file) $(citus_abs_srcdir)/regression.diffs)
+	$(pg_regress_multi_check) --vanillatest || (cp $(vanilla_diffs_file) $(citus_abs_srcdir)/regression.diffs && false)
```

Comment:
I'm assuming you checked that CI fails after this change and without the others?

Reply:
yep.